### PR TITLE
fix: mouse snapping in settings

### PR DESCRIPTION
**Pull request by JoenvandeVorle**
## Description

The first time when clicking on something in the menu, the cursOr used to snap back to the center of the screen. This no longer happens. 

## Testability

Go to the `Settings` or `InGameSettings` scene and press escape once in-game. This will open the menu, and you can check for yourself. 
### **NOTE:** 
Going to the menu is bound to escape. In the editor, this will unfocus you from the game window, which means the cursor will stay visible if you try to close the settings by pressing escape again. This does not happen when you build and run.

## Checklist
- [x] Set the proper pull request name
- [x] Merged main into your branch
- [x] Added a scene to the game for your changes

### CODE REVIEW

**Review by SSelinn — APPROVED** [review]
Looks good! Almost snapped without this.

**Review by SilasPeters — CHANGES_REQUESTED** [review]
Exiting the settings through `esc` does not lock the cursor. Pressing quit does not do anything. Tested in `InGameSettings` scene. Also, why does it say `Quit` instead of `Quit game`, like in `Settings` scene?

File: aplib.net-demo/Assets/Input/PlayerInput.cs [hunk 1]
```diff
@@ -1324,6 +1358,9 @@ public void AddCallbacks(IUIActions instance)
             @ShowMouse.started += instance.OnShowMouse;
             @ShowMouse.performed += instance.OnShowMouse;
             @ShowMouse.canceled += instance.OnShowMouse;
+            @OpenSettings.started += instance.OnOpenSettings;
+            @OpenSettings.performed += instance.OnOpenSettings;
+            @OpenSettings.canceled += instance.OnOpenSettings;
```
[the remark this answers is not in the document]
Comment (JoenvandeVorle) [reply aplib.net-demo/Assets/Input/PlayerInput.cs:1363]: The mysteries of the Unity Input System are not for us to mere mortals to uncover Silas

File: aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs [hunk 2]
```diff
@@ -52,7 +55,15 @@ public class CanvasManager : MonoBehaviour
     /// Name of the game scene
     /// </summary>
     [SerializeField]
-    private string _sceneNameGame = "MeleeWeaponMenu"; // TODO:: Load main game screen
+    private string _sceneNameGame = "GameSettings"; // TODO:: Load main game screen
```
[the remark this answers is not in the document]
Comment (JoenvandeVorle) [reply aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs:58]: Well, the main game scene does not exist yet

**Comment by JoenvandeVorle** [comment]
> Exiting the settings through `esc` does not lock the cursor. Pressing quit does not do anything. Tested in `InGameSettings` scene. Also, why does it say `Quit` instead of `Quit game`, like in `Settings` scene?

Have you read the Note under testability?

**Comment by SilasPeters** [comment]
> Have you read the Note under testability?

Absolutely not

**Review by SilasPeters — APPROVED** [review]

**Pull request merged**